### ISSUE TITLE
[Craft 4] renderForm method throws hard error when passing non-existing form handle

### ISSUE THREAD

**Issue by Antimated** [verbb/formie#938]
**Description**
In the past you could pass a handle that doesn't exist to the renderForm method (which would just silently error).
Now when I do this I get a hard error:

![formie-error](https://user-images.githubusercontent.com/2869409/174020004-3d60dff2-8a61-453a-a9fb-67ff58fb805c.png)

I'd rather have the old behaviour where it just silently errors. Imagine adding a `newsletter` form like so `{{ craft.formie.renderForm('newsletter', { formClasses: ['form'] }) }}` on a local environment. Then you push out your changes to a staging or production environment where a form with that handle is not yet present. In the current state the website would just throw a hard server error until you add a form with the handle `newsletter`.

**Steps to reproduce**
1. Add the following code to your site without making a form with the handle `newsletter`: `{{ craft.formie.renderForm('newsletter', { formClasses: ['form'] }) }}`
2. Inspect error

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.13
- Craft version: Craft Pro 4.0.4
- Multi-site: Yes

**Comment by engram-design** [comment]
I'm not really keen on changing this back, because it doesn't error (even silently) when the form doesn't exist, leading to people sending support tickets of "why isn't my form showing?!" - because it doesn't exist.

But I suppose I may have to add it back in, because you can pass in a string to render. Normally, I'd say code defensively and check if the form exists before you render it.

**Comment by engram-design** [comment]
You've twisted my arm. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by Antimated** [comment]
> You've twisted my arm.


Sorry about the arm 😀

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/formie/releases/tag/2.0.0)
